streaming/base/converters/dataframe_to_mds.py

```diff
@@ -287,6 +291,16 @@ def write_mds(iterator: Iterable):
 
     # Fix output format as mds_path: Tuple(local, remote)
     if cu.remote is None:
+        # If dataframe_to_mds is being called, this is in a distributed Spark env.
+        # If there is no remote, it's because the given out path is local, which does not
+        # in general make sense, unless this 'local' path is FUSE-mounted distributed
+        # storage such as /dbfs or /Volumes in Databricks for example.
+        # It's not wrong in this case, but probably nevertheless desirable to specify a local temp
+        # path explicitly, to interpret the FUSE-mounted path as remote
+        logger.warning(f"Path {cu.local} is interpreted as a local path. If this is actually " +
+                       'mounted distributed storage, it will work, but consider also specifying ' +
```

Comment:
```suggestion
                       'mounted distributed storage it will work, but consider also specifying ' +
```

Reply:
This comma seems correct, even necessary

Reply:
Same as above, very minor and doesn't matter to me